scripts/repair_wheel.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import re
+import shutil
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+from convert_to_generic_platform_wheel import convert_to_generic_platform_wheel
+
+
+def main():
```

Comment:
```suggestion
def main() -> None:
```

Reply:
flake8 is failing on this one when ran from python2. I'd rather let this be for now unless you have a simple solution.

Reply:
Fine for now. We should change to using pre-commit. I can see to that soonish.